### PR TITLE
Connect with apple

### PR DESCRIPTION
**Pull request by IdoMGini**

### CODE REVIEW

File: GASocialLogin/Apple/GAAppleLoginService.swift [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+//
+//  Created by arkadi yoskovitz on 07/07/2020.
+//
+
+import Foundation
+import AuthenticationServices
+
+@available(iOS 13.0, *)
+public typealias GAAppleUserCredential  = ASAuthorizationAppleIDCredential
+public typealias GAAuthorizationKeys    = ASAuthorization.Scope
```
Comment (ArkadiGiniApps) [inline GASocialLogin/Apple/GAAppleLoginService.swift:14]: this should have a separate @ available marking

File: GASocialLogin/Apple/GAAppleLoginService.swift [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            controller.delegate = self
+            
+            if viewController != nil {
+                
+                controller.presentationContextProvider = self
+            }
+            
+            controller.performRequests()
+            
+            self.authorizationController = controller
```
Comment (ArkadiGiniApps) [inline GASocialLogin/Apple/GAAppleLoginService.swift]: do we really need self here?

**Review by ArkadiGiniApps — CHANGES_REQUESTED** [review]

File: GASocialLogin/Apple/GAAppleLoginService.swift (hunk 1 above)
Comment (IdoMGini) [reply GASocialLogin/Apple/GAAppleLoginService.swift:14]: done

File: GASocialLogin/Apple/GAAppleLoginService.swift (hunk 2 above)
Comment (IdoMGini) [reply GASocialLogin/Apple/GAAppleLoginService.swift]: done

**Pull request merged**